modules/oxide/src/less/theme/content/contenteditable/contenteditable.less

```diff
@@ -33,9 +33,14 @@
 
   *[contentEditable=false] {
     cursor: default;
+    user-select: none;
   }
 
   *[contentEditable=true] {
     cursor: text;
   }
+
+  *[contentEditable=false] *[contentEditable=true] {
```

Comment:
Couldn't this just go in the `*[contentEditable=true]` block? `user-select` is not inherited so it should still work the same way `cursor` does.

- [ ] consider moving `user-select` into the `true` block

Reply:
It seems there is an exception for Safari: https://fiddle.tiny.cloud/yiiaab/7
Without  `*[contentEditable=false] *[contentEditable=true] `rule  I can not select content of nested `cet` element

Reply:
I can select text in both strong tags, but I'm running Safari 16 beta. I guess it'll be another year or so before we can rely on that. Maybe leave a note about it in a comment? Doesn't seem worth making a task for.